lib/versioning/maven/compare.js

```diff
@@ -218,10 +218,9 @@ function compare(left, right) {
 function isPrimitiveVersion(version) {
   // istanbul ignore if
   if (!version) return false;
-  if (!/^v?\d[a-z0-9.-]*$/i.test(version)) return false;
+  if (!/^[a-z0-9.-]+$/i.test(version)) return false;
```

Comment:
Does this mean a version can start with a `.` or `-`?  Should we tighten it so that it must start with letter/digit first and then have those?

Reply:
Oh, i'll fix it